dynawo/sources/Solvers/Common/DYNSolver.h

```diff
@@ -47,7 +47,8 @@ typedef enum {
   ModeChange = 0x01,
   NotSilentZChange = 0x02,
   SilentZNotUsedInDiscreteEqChange = 0x04,
-  SilentZNotUsedInContinuousEqChange = 0x08
+  SilentZNotUsedInContinuousEqChange = 0x08,
+  NRSkipped = 0x10
 } StateFlags;
```

Comment:
I think it should'nt be part of this enum as it encompasses a quite different meaning.
The enum is currently related to event handling scheme and to what is going on in terms of discrete variables and mode changes and so integrating here the NRSkipped doesn't seem the best option in my opinion.

I would definetely prefer to have a specific boolean or a specific status than reusing the existing one.

Reply:
even if it means duplicating the mechanism ?